src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -122,16 +122,19 @@
                 }
 
                 toggle.addEventListener('click', (e) => {
+                    const toggle = e.currentTarget;
+
+                    if (e.target.closest('a, .sf-toggle') !== toggle) {
```

Comment:
should we handle buttons inside toggles as well ?

Reply:
I guess `button`s and `type="button"` `input`s could be handled the same way 🤔